frontend/coprs_frontend/coprs/models.py

```diff
@@ -1812,9 +1819,20 @@ def delete_status(self):
 
             return ChrootDeletionStatus("preserved")
 
+        # Normal enabled && active chroots
         if not self.delete_after and not self.delete_notify:
             return ChrootDeletionStatus("active")
 
+        # EOLed rolling chroots (reactivate by a new build)
+        if self.delete_after and self.mock_chroot.rolling:
+            # We can never ever remove EOL chroots that we didn't send
+            # a notification about
+            if not self.delete_notify:
+                return ChrootDeletionStatus("preserved")
+            if self.delete_after < datetime.datetime.now():
+                return ChrootDeletionStatus("expired")
```

Comment:
The "preserved" chroots are never removed, so we require an e-mail notification here, and return "expired".